etc/examples/caffe-model/gpu-manifest.yml

```diff
@@ -2,9 +2,9 @@ name: mnist-caffe-gpu-model
 description: Caffe MNIST model (GPU)
 version: "1.0"
 gpus: 1
-cpus: 2
+cpus: 0.005
```

Comment:
The cpus setting is from the old example. we should change it back to 2 for the sample job to be more efficient.